ui/src/duck/auth.ts

```diff
@@ -28,6 +36,12 @@ export const fetchCurrentUser = createAsyncThunk<
   { rejectValue: Error; state: AuthState }
 >('auth/fetchCurrentUser', apiFetchCurrentUser);
 
+export const sendPasswordResetOTP = createAsyncThunk<
+  undefined,
+  SendOTPPayload,
+  { rejectValue: Error; state: PasswordResetOTP }
```

Comment:
`createAsyncThunk` gets the state of the slice. So here it should be `AuthState` and not `PasswordResetOTP`.